facia-purger/src/main/scala/com/gu/purge/facia/FrontsS3PathParser.scala

```diff
@@ -4,8 +4,7 @@ import org.parboiled2.{ Parser, ParserInput }
 
 import scala.util.Success
 
-// /aws-frontend-store/CODE/frontsapi/pressed/live/au/fapi/pressed.json
-class ParseS3Path(stage: String, val input: ParserInput) extends Parser with Logging {
+class FrontsS3PathParser(stage: String, val input: ParserInput) extends Parser with Logging {
```

Comment:
although having said that, the tests provide that documentation in executable form, however in a less convenient location